packages/server/api/src/app/ee/flow-template/platform-flow-template.module.ts

```diff
@@ -105,8 +105,7 @@ async function resolveTemplatesPlatformId(principal: Principal): Promise<string
 
 const GetParams = {
     config: {
-        allowedPrincipals: ALL_PRINCIPAL_TYPES,
-        scope: EndpointScope.PLATFORM,
+        security: publicAccess(),
```

Comment:
Sadly these are not the same, publicAcess actually doesn't have authenticated, It keeps the prinicpal null.

In older case It does authenticate but doesn't authorize